project-base/composer.json

```diff
@@ -98,7 +84,6 @@
         "symfony/symfony": "^3.4.8",
         "symfony-cmf/routing": "^2.0.3",
         "symfony-cmf/routing-bundle": "^2.0.3",
-        "timwhitlock/jparser": "@dev",
```

Comment:
Reason is that the `jparser` is used only in `framework` package. It had to be here because of `dev-master` dependecy from `framework`. Since the `jparser` is now required in version `0.1`, we don't need to require it also here in project-base. Should I explain this in commit message?